### PR TITLE
Override feature flags, bundle id, and app name

### PR DESCRIPTION
**Pull request by ps2**

### CODE REVIEW

File: LoopConfigOverride.xcconfig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+//
+
+MAIN_APP_BUNDLE_IDENTIFIER = com.tidepool.org
```
Comment (darinkrauss) [inline LoopConfigOverride.xcconfig]: com.tidepool.org?

Comment (darinkrauss) [reply LoopConfigOverride.xcconfig]: org.tidepool.loop instead?

Comment (ps2) [reply LoopConfigOverride.xcconfig]: :) Thanks.

**Comment by ps2** [comment]
Merging plists was the first iteration of overridable feature flags, then I switched to an xcode configuration based version. The LoopWorkspace referencing the `Merge Feature Flags` target was leftover cruft. Looks like it didn't break the build, but is good to clean up. Thanks for catching it.

**Pull request merged**